### ISSUE TITLE
Not working on iOS

### ISSUE THREAD

**Issue by fvdm** [aonez/GitHubCounter#3]
On iOS it is loading and the yellow triangle appears in the Safari extensions to allow the app access to github.com. But after granting access the counters do not appear on the release assets.

I tested on my own public repos and the Keka repo from the screenshot in the readme.

- iOS 17.4.1
- Ghounter 2.1.0

Test: https://github.com/aonez/Keka/releases/tag/v1.2.18

![image](https://github.com/aonez/GitHubCounter/assets/93784/f4fc1b48-feab-484a-9f3b-046a7784c898)

**Comment by aonez** [comment]
Does it work if you refresh the web? It does for me.

**Comment by fvdm** [comment]
Yeah, I tried refresh, clearing the Safari cache, reinstall. But now I'm looking at the Github API response and see the `assets` array is empty for all files. So I think there may be something off overthere.

**Comment by aonez** [comment]
There’s a limit in the calls you can make to the API. Does ot work if you
try it in a private browser tab without any GitHub user?
